mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/tilsagn/db/TilsagnQueries.kt

```diff
@@ -56,7 +58,8 @@ class TilsagnQueries(private val session: Session) {
                 arrangor_id             = excluded.arrangor_id,
                 beregning               = excluded.beregning,
                 status                  = excluded.status,
-                type                    = excluded.type
+                type                    = excluded.type,
+                gjenstaende_belop       = excluded.gjenstaende_belop
```

Comment:
Kan vi kalle denne kolonnen for `belop_gjenstaende`? Jeg tenkte å legge til en `belop_beregnet`-kolonne ifm. normalisering av beregning :grin: Evt. kan jeg gjøre en refaktorering i neste pr, om dere er enig i navngiving

Reply:
Ja, det kan vi

Reply:
Jeg oppdaterer